Source/EasyNetQ.Management.Client/IManagementClient.cs

```diff
@@ -702,4 +702,18 @@ public interface IManagementClient : IDisposable
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
     Task RebalanceQueuesAsync(CancellationToken cancellationToken = default);
+
+    /// <summary>
+    ///     Creates a shovel in a specific vhost
+    /// </summary>
+    /// <param name="vHostName"></param>
+    /// <param name="name"></param>
+    /// <param name="parameters"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task CreateShovelAsync(
+        string vHostName,
+        string name,
+        ParameterShovelValue parameters,
+        CancellationToken cancellationToken = default);
```

Comment:
Sorry, dotnet format doesn't handle this well automatically, could you move `)` to the new line here and in other places you added?

Reply:
Done